alias config

RaghvindYadav opened the pull request:
https://svelte.dev/docs/kit/configuration#alias

<!-- This template is to serve as a comprehensive outline for documenting updates to the code. Some of these sections may not be relevant to your PR. In that case, document it inline with `N/A`  -->

## What does this PR do?
While reading the official documentation, I found the [alias](https://svelte.dev/docs/kit/configuration#alias) method would be the best method to import the files.

My primary goal with this PR was to standardize our import methods, as we were inconsistently using both @ and $ symbols for importing components.


## Final Checklist
- [x] Linked related issues
- [x] Rebased with target branch
- [x] No lint errors
- [x] QA'd in multiple browsers
- [x] Added/updated tests (if applicable)
- [x] Mobile responsive (if applicable)
- [x] Updated documentation (if applicable)
- [x] Notified the Team of this updates readiness

---

## AI Usage
<!-- Document any AI tools you used to develop this -->
- [ ] AI tools were used in development
  - **Tool used**: No any AI tool is used.
  - **Prompts used**:
    <!-- Format these inclusions as quotes, like: `> Provide  -->
    - **Initial prompt**: 
- [ ] No AI tools were used
  - **Reason**:
    Just need to config the files and update import method and I found it easier to do without AI.

Commit:
alias config

https://svelte.dev/docs/kit/configuration#alias

Commit:
update import

Commit:
Update +layout.ts

jonahtwalker commented on src/routes/reset-password/reset-password-form.svelte:
```diff
@@ -357,7 +357,6 @@
       {/if}
       <FormButton
         disabled={$delayed || isLoading || hasStepError}
-        loading={$delayed || isLoading}
         buttonText={getCurrentStepButtonLabel()}
```
@RaghvindYadav What about this?

jonahtwalker commented on src/routes/signin/sign-in-form.svelte:
```diff
@@ -378,7 +378,6 @@
         <FormButton
           disabled={$delayed || isLoading || hasStepError}
-          loading={$delayed || isLoading}
           buttonText="Sign in"
```
@RaghvindYadav What about this?

jonahtwalker commented on src/routes/signup/sign-up-form.svelte:
```diff
@@ -442,7 +442,6 @@
       {/if}
       <FormButton
         disabled={$delayed || isLoading || hasStepError}
-        loading={$delayed || isLoading}
         buttonText="Sign Up"
```
@RaghvindYadav What about this?

RaghvindYadav replied:
It was coming from milestone-01, because loading parameter was changed to isLoading. I thought it was removed soI. remove the code it. Let me replace the variable with isLoading.

jonahtwalker reviewed (changes requested):
Please fix the form button.

What prompted this change for aliases?

Commit:
loading variable changed

RaghvindYadav commented:
> What prompted this change for aliases?

There were a few errors in the milestone related to the loading field because the variable name had been changed to isLoading. I initially assumed the loading parameter was no longer needed, so I removed the related code.

However, it's now fixed—I updated the code to use the correct variable name.

jonahtwalker commented:
Sorry Raghvind, I don't think this is the correct approach. We do need to update imports, but we do not need to create more aliases just to save a single `/` on the imports. We already have the `@` alias that is used to import from the `lib` directory, and is just as much of a shortcut.

```
import { myUserContext } from '$contexts/my-user-context.svelte';
import { myUserContext } from '@/contexts/my-user-context.svelte';
```

Further, the correct imports for components would be:
```
import * as Avatar from '$components/ui/avatar';
import { Button } from '@/components/ui/button';
```

Please remove the new aliases and normalize the imports. 👍🏼

RaghvindYadav commented:
> ```
> import * as Avatar from '$components/ui/avatar';
> import { Button } from '@/components/ui/button';
> ```

Okay, I will do, should we use two different symbols for import?

RaghvindYadav commented:
I think this not relevant.
